packages/desktop/components/popups/send/SendConfirmationPopup.svelte

```diff
@@ -71,6 +77,7 @@
     $: isLayer2Transaction = !!layer2Parameters
     $: isTransferring = $selectedWallet.isTransferring
     $: hideGiftToggle = isBaseTokenTransfer || isLayer2Transaction || (disableToggleGift && !giftStorageDeposit)
+    $: requiredMana = allotmentManaCost + extraMana
```

Comment:
```suggestion
    $: requiredMana = Number(allotmentManaCost) + extraMana
```
i'm suggesting this change because for some reason, `allotmentManaCost` changes from being a string to a number
![image](https://github.com/iotaledger/firefly/assets/72078237/91526d0e-6137-42c6-825d-9d3e83784ec2)

and there's a weird flash:
https://github.com/iotaledger/firefly/assets/72078237/43871bed-973b-428a-986a-529d700a2b92